src/emulator/CMakeLists.txt

```diff
@@ -34,12 +34,17 @@ load_self.h
 main.cpp
 relocation.cpp
 relocation.h
+resource.h
+VIta3K.ico
+Vita3K.rc
 vpk.cpp
 vpk.h
 )
 
 target_link_libraries(emulator PRIVATE cpu elfio host modules nids vita-toolchain)
-set_target_properties(emulator PROPERTIES OUTPUT_NAME Vita3K)
+set_target_properties(emulator PROPERTIES OUTPUT_NAME Vita3K
+    ARCHIVE_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/bin"
+    RUNTIME_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/bin")
```

Comment:
What happens if we don't change the output directory?

Reply:
bad folder, compiled on "build-windows\src\emulator\(Release,Debug)" like that move on "build-windows\bin\(Release,Debug)"

Reply:
OK. If it's clearer and works for everyone that's good. Thanks!